R/batch.R

```diff
@@ -16,8 +16,10 @@
 batch <- function(batch_fn, keys, splitting_strategy = NULL,
   combination_strategy = batchman::combine, size = 50, verbose = TRUE,
   trycatch = FALSE, stop = FALSE) {
-    if (is.batched_fn(batch_fn)) return(batch_fn)
+    if (isTRUE(attr(batch_fn, 'batched'))) return(batch_fn)
```

Comment:
Is it in the NAMESPACE file?

On Wed, Mar 25, 2015 at 12:45 PM, Peter Hurford <notifications@github.com>
wrote:

> In R/batch.R
> <https://github.com/peterhurford/batchman/pull/25#discussion_r27146350>:
>
> > @@ -16,8 +16,10 @@
> >  batch <- function(batch_fn, keys, splitting_strategy = NULL,
> >    combination_strategy = batchman::combine, size = 50, verbose = TRUE,
> >    trycatch = FALSE, stop = FALSE) {
> > -    if (is.batched_fn(batch_fn)) return(batch_fn)
> > +    if (isTRUE(attr(batch_fn, 'batched'))) return(batch_fn)
>
> This is lame -- somehow R now insists that is.batched_fn is no longer
> exported. Can't figure out why. Hasn't happened in other PRs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/peterhurford/batchman/pull/25/files#r27146350>.
>



-- 
-Robert Krzyzanowski